src/widgets/PaperModal/PaperConfirmModal.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { StyleSheet, Text } from 'react-native';
+import PropTypes from 'prop-types';
+import { FlexColumn } from '../FlexColumn';
+import { FlexView } from '../FlexView';
+import { HazardIcon } from '../icons';
+import { Spacer } from '../Spacer';
+import { DARKER_GREY, SUSSOL_ORANGE, WHITE } from '../../globalStyles/index';
+import { FlexRow } from '../FlexRow';
+import { PageButton } from '../PageButton';
+
+export const PaperConfirmModal = ({
+  Icon,
+  questionText,
+  confirmText,
+  cancelText,
+  onConfirm,
+  onCancel,
+}) => (
+  <FlexColumn alignItems="center" justifyContent="center" flex={1} style={{ padding: 20 }}>
+    <FlexView flex={1} alignItems="center" justifyContent="center">
+      {Icon}
+      <Spacer space={30} vertical />
+      <Text style={localStyles.questionText}>{questionText}</Text>
+    </FlexView>
+    <FlexRow justifyContent="space-evenly" alignItems="center" style={{ width: '100%' }} flex={1}>
+      <PageButton onPress={onCancel} text={cancelText} textStyle={{ textTransform: 'uppercase' }} />
+      <PageButton
+        onPress={onConfirm}
+        text={confirmText}
+        textStyle={{ textTransform: 'uppercase', color: WHITE }}
+        style={{ backgroundColor: SUSSOL_ORANGE }}
+      />
+    </FlexRow>
+  </FlexColumn>
+);
```

Comment:
somewhat reuseable. 

I'm sure it'll need changes when it is actually reused 🤣